test/components/connect.spec.js

```diff
@@ -332,12 +332,20 @@ describe('React', () => {
       expect(decorated.state.data.testFunc).toBeA('function')
       expect(decorated.state.data.deferredFetch).toEqual(undefined)
 
-      decorated.state.data.testFunc('A', 'B')
+      decorated.state.data.testFunc('A', 'B').then(value => expect(value).toEqual({ T: 't' }))
```

Comment:
...also, this assertion looks wrong. shouldn't it be something like `value.deferredFetch.value`?